script/cypress-testrail-helper/lib/app-inquirer.js

```diff
@@ -63,83 +63,4 @@ module.exports = {
 
     return inq.prompt(questions);
   },
-
-  askTestRailRunOptions() {
```

Comment:
For this file, I'm just deleting two unrelated methods (lines 66-144) that were no longer used.